db/migrate/20210318034405_add_application_status_to_teachers.rb

```diff
@@ -0,0 +1,17 @@
+class AddApplicationStatusToTeachers < ActiveRecord::Migration[5.2]
+  def change
+    add_column :teachers, :application_status, :string, :default=>"Pending"
+    Teacher.all.each do |t|
```

Comment:
This seems to work in this app, though in general it's a best practice to keep data manipulation out of the migration. 

This migration is also unfortunately not quite right -- though I didn't catch this before deploying. There were no teachers were `denied == false` because the previous migration didn't set a default. It might have been safer to just write `t.validated and not t.denied`